References: bsc#1180897
Removing the cdrom iso file from the device prevents booting
with a qemu error.

Index: virt-manager-5.1.0/virtinst/install/installer.py
===================================================================
--- virt-manager-5.1.0.orig/virtinst/install/installer.py
+++ virt-manager-5.1.0/virtinst/install/installer.py
@@ -184,8 +184,9 @@ class Installer:
             return
         for disk in guest.devices.disk:
             if disk.is_cdrom() and disk.get_source_path() == self._cdrom_path():
-                disk.set_source_path(None)
-                disk.sync_path_props()
+                if not guest.os.is_xenpv():
+                    disk.set_source_path(None)
+                    disk.sync_path_props()
                 break
 
     def _add_unattended_install_cdrom_device(self, guest, location):
